src/main/kotlin/calculator/Operator.kt

```diff
@@ -1,21 +1,26 @@
 package calculator
 
-enum class Operator(val symbol: String) {
-    PLUS("+"),
-    MINUS("-"),
-    MULTIPLY("*"),
-    DIVIDE("/");
+enum class Operator(val symbol: String, private val operation: IntOperation) {
+    PLUS("+", IntOperation { number1, number2 -> number1 + number2 }),
+    MINUS("-", IntOperation { number1, number2 -> number1 - number2 }),
+    MULTIPLY("*", IntOperation { number1, number2 -> number1 * number2 }),
+    DIVIDE("/", IntOperation { number1, number2 -> number1 / number2 });
```

Comment:
SAM 활용 👍💯

조금 더 간단하게 아래와 같이 변경해볼 수도 있을 것 같아요. 🤗
```suggestion
    PLUS("+", { number1, number2 -> number1 + number2 }),
    MINUS("-", { number1, number2 -> number1 - number2 }),
    MULTIPLY("*", { number1, number2 -> number1 * number2 }),
    DIVIDE("/", { number1, number2 -> number1 / number2 });
```